extensions/ql-vscode/package.json

```
@@ -969,6 +969,8 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "semver": "~7.3.2",
+    "stream-chain": "^2.2.4",
+    "stream-json": "^1.7.3",
```

nit: For consistency:

```suggestion
    "stream-chain": "~2.2.4",
    "stream-json": "~1.7.3",
```